docs/1.concepts/account-abstraction/chainsig.md

```diff
@@ -0,0 +1,114 @@
+---
+id: chain-signatures
+title: Chain Signatures
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+Chain Signatures are a method to allow a smart contract to call an async method. For example:
+
+```rust
+async fn sign(
+    payload: Bytes,
+    key_derivation: Bytes,
+    key_type: ECDSA) -> Signature
+```
+
+The signature is calculated on the data payload using a user-provided key. The signature output is always the same, as long as the caller and payload remains unchanged.
```

Comment:
Interestingly the signatures aren't actually the same, they're different but both valid.
```suggestion
The signature is calculated on the data payload using a user-provided key. The signature's public key is always the same, as long as the caller and the key path remain the same.
```